diff -Nur kwappen-1.1.0.orig/Makefile.am kwappen-1.1.0/Makefile.am
--- kwappen-1.1.0.orig/Makefile.am	Mon Jun 10 22:48:59 2002
+++ kwappen-1.1.0/Makefile.am	Mon Jun 10 22:49:25 2002
@@ -2,7 +2,7 @@
 
 SUBDIRS = kwappen po doc 
 
-EXTRA_DIST = kwappen.kdevprj AUTHORS COPYING ChangeLog INSTALL README TODO kwappen.lsm kwappen.spec 
+EXTRA_DIST = kwappen.kdevprj AUTHORS COPYING ChangeLog INSTALL README TODO kwappen.lsm kwappen.spec admin
 
 ####### kdevelop will overwrite this part!!! (end)############
 # not a GNU package. You can remove this line, if 
@@ -24,8 +24,6 @@
 	$(MAKE) -f admin/Makefile.common package-messages
 	$(MAKE) -C po merge
 
-EXTRA_DIST = admin 
-
 dist-hook:
 	cd $(top_distdir) && perl admin/am_edit -padmin
 	cd $(top_distdir) && $(MAKE) -f admin/Makefile.common subdirs

diff -Nru kwappen-1.1.2.orig/po/Makefile.am kwappen-1.1.2/po/Makefile.am
--- kwappen-1.1.2.orig/po/Makefile.am	Sat Dec 28 15:36:01 2002
+++ kwappen-1.1.2/po/Makefile.am	Sat Dec 28 15:53:18 2002
@@ -1,6 +1,5 @@
 ####### kdevelop will overwrite this part!!! (begin)##########
-POFILES = de.po 
-
+EXTRA_DIST = kwappen.pot
 
 ####### kdevelop will overwrite this part!!! (end)############
 POFILES = AUTO

